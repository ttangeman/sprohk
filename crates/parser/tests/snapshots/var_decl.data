NodeData {
    var_decls: [
        VarDecl {
            specifier: Var,
            name: 1,
            type_spec: Some(
                3,
            ),
        },
    ],
}