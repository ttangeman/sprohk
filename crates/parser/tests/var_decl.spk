var foo;
let bar: i32;
const foobar: i32 = 42;