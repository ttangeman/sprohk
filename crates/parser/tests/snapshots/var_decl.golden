node 0 {
  kind: VarDecl
  tokens: [0..3]
  data: {
    specifier: Var
    name: 'foo'
    type_spec: 'i32'
  }
}
